includes/restoreMappings.js

```diff
@@ -65,14 +66,24 @@ class Restore {
         return lineAsJson;
       } else if (backupLine.lineNumber === 1 && lineAsJson.name && lineAsJson.version && lineAsJson.mode) {
         // First line is metadata.
-        mappingDebug(`Parsed backup file metadata ${lineAsJson.name} ${lineAsJson.version} ${lineAsJson.mode}.`);
+        mappingDebug(`Parsed backup file metadata ${lineAsJson.name} ${lineAsJson.version} ${lineAsJson.mode} ${lineAsJson.attachments}.`);
         // This identifies a version of 2.10.0 or newer that wrote the backup file.
         // Set the mode that was used for the backup file.
         this.backupMode = lineAsJson.mode;
         // For newer versions we don't need to ignore all broken JSON, only ones that
         // were associated wiht a resume, so unset the ignore flag.
         this.suppressAllBrokenJSONErrors = false;
         // Later we may add other version/feature specific toggles here.
+        if (lineAsJson.attachments === true) {
+          if (!this.options.attachments) {
+            // Error out if trying to restore attachments without the option
+            throw new BackupError('AttachmentsNotEnabledError', 'To restore a backup file with attachments enable the attachments option.');
```

Comment:
What would be the workaround if they backed up a database with attachments on Couch, and they want to restore it on a Cloudant instance where attachment option is not allowed? Can they just remove that part of the metadata in the backup file manually? If so, maybe it worth it to mention in the documentation.

Reply:
>What would be the workaround if they backed up a database with attachments on Couch, and they want to restore it on a Cloudant instance where attachment option is not allowed? 

Well this would be an unsupported use case. That said nothing forbids it, people can still use `-a` with their Cloudant instance, we just won't provide support if it goes wrong (and it is likely "expensive" because of write quota usage).

>Can they just remove that part of the metadata in the backup file manually?

Yes, but it'd be easier just to pass `-a`.

Reply:
Thanks! So although they get the warning (`Do not use this option with IBM Cloudant`), they can still use it with Cloudant without guarantees it works fine.

Can you put that comma there that I suggested [in the first comment](https://github.com/IBM/couchbackup/pull/760#discussion_r1738498067) of this conversation?

Reply:
Right, as an experimental option we're not going to guarantee it works anyway, but we're going to specifically discourage Cloudant users from doing attachments because of cost to them. Using the DB to store binary blobs is not cost effective compared to e.g. COS and restoring backups with attachments in requests is also capacity intense. Anyone willing to accept the risk and cost can still do it.

Sorry I missed the comma fixed in 55401f2ae5ece4ac6da107712738715424d9a53c